app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+
+### 3. Date of arrival 
+
+You must move to England no more than 3 months before the start of your job.  
+
+### Non-eligibility for teachers 
+
+You will not be eligible for the international relocation payment if you are: 
+
+* a UK citizen 
+
+* an Irish citizen 
+
+* a non-UK citizen who moved to England to live permanently more than 3 months before the start of your job 
+
+* teaching only English language and literature
+
+* employed on a contract lasting less than one academic year 
+
+* employed by a recruitment agency 
+
+## How teachers can apply for the international relocation payment 
+
+You’ll need to have started your teaching job in a state secondary school to make your application for the international relocation payment (IRP). [Learn more about finding a teaching job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
+
+You will be able to apply for the IRP from 1 September to 31 October 2023. You can email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open. If you are eligible, you’ll receive the money around the end of your first school term of employment. 
+
+For information about IRP applications in 2024, check back to this page. 
+
+## General information about the international relocation payment  
+
+Further information for trainees and teachers
+
+### Protect yourself from fraud 
+
+Teachers and salaried trainee teachers will not need to pay an agent or any other third party to complete their application. Any agency or person contacting you offering to apply for the IRP on your behalf is likely to be fraudulent. [Avoid and report internet scams and phishing](https://www.gov.uk/report-suspicious-emails-websites-phishing). 
+
+### Visiting England more than 3 months before the start of your course or job 
+
+Your eligibility for the international relocation payment (IRP) will not be affected if you spend short periods in England at a time more than 3 months before the start of your course or job. 
+
+Example 1: You attend a job interview in England under a Standard Visitor visa in May 2023. You return home, and learn you have been given the job, which starts in September 2023. You apply for a Skilled worker visa in June 2023, 3 months before the start of your job. You move to England in July 2023. You will still be eligible for the IRP. 
+
+Example 2: You have a UK Ancestry visa and use it to visit relations in England in February 2023. You return home and successfully apply for a place on a teacher training course starting in September 2023. You move to England in June 2023. You will still be eligible for the IRP. 
+
+You will also still be eligible for the IRP if you live in another part of the UK (for example, Scotland or Wales) as long as you are employed in a school in England.  
```

Comment:
```suggestion
You will also still be eligible for the IRP if you're a non-UK citizen living in another part of the UK (for example, Scotland or Wales) as long as you are employed in a school in England.  
```
Do we need a bit of clarification here, so that Scottish/Welsh citizens don't think they're eligible for the IRP? (Unless they are?)

Reply:
Yes good point